packages/database/src/migrations/20230413041856-FixupDropAnalyticsLogTablesFunction-modifies-schema.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function (db) {
+  await db.runSql(`
+  CREATE OR REPLACE FUNCTION drop_analytics_log_tables() RETURNS void AS $$
```

Comment:
Shall we pop this in `migrations/migrationData/dropAnalyticsLogTables/createDropAnalyticsLogTablesFunction.v1.js` like we do for buildAnalyticsTableFunction?

Reply:
Yeah good call 👍